CameraOverlay/overlay.py

```diff
@@ -241,5 +243,8 @@ def update_data_layer(self):
 	@staticmethod
 	def get_overlay_args(overlay_description: str):
 		parser = argparse.ArgumentParser(description=overlay_description, add_help=True)
-		parser.add_argument("--host", action="store", type=str, default="localhost", help="Address of the MQTT broker")
+		parser.add_argument("--host", action="store", type=str, default="localhost",
+			help="Address of the MQTT broker")
+		parser.add_argument("-b", "--bike", action="store", type=str,
+			choices=["v2", "V2", "v3", "V3"], help="Specify the which bike to expect MQTT data from")
```

Comment:
DataFactory might break with lowercase. Update factory to not be case sensitive.